### PR TITLE
Fix particle system build errors

### PR DESCRIPTION
**Pull request by eli-lam**
Caught a bug in the built version with the particle system. Fixed.

### CODE REVIEW

**Review by jmcduff — APPROVED** [review]

File: Assets/Scripts/Combat/Abilities/FireBall.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ private enum FireballCyclePhase { Normal, UnstableWarning, Unstable };
         private const float FireballHeight = 7f;
         private const float FireballScale = 0.08f;
         private float TargetFireballSize = 0.15f;
-        private const float FireballGrowth = 0.025f;
+        private const float FireballGrowth = 2f;
```
Comment (danielwiktorczyk) [inline Assets/Scripts/Combat/Abilities/FireBall.cs:31]: Woah woah woah why

**Review by danielwiktorczyk — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Combat/Abilities/FireBall.cs (hunk 1 above)
Comment (eli-lam) [reply Assets/Scripts/Combat/Abilities/FireBall.cs:31]: Using delta time now

Comment (danielwiktorczyk) [reply Assets/Scripts/Combat/Abilities/FireBall.cs:31]: image
The size increase now is... even bigger

**Review by danielwiktorczyk — CHANGES_REQUESTED** [review]

**Comment by danielwiktorczyk** [comment]
So yah, revert the scale, and remove Time.deltaTime pls. Jump on VC to remind me to approve!

**Review by danielwiktorczyk — APPROVED** [review]
Merci!

**Review by armandorusso — APPROVED** [review]
Went over it with Daniel in a call. Fixes look good!

**Pull request merged**